.circleci/config.yml

```diff
@@ -26,9 +26,16 @@ build_step: &build_step
 test_step: &test_step
   run:
     name: Test
-    command: test -z "$LLDB_TESTS" || (cd build && ../Support/Scripts/run-lldb-tests.sh)
+    command: cd build && ../Support/Scripts/run-lldb-tests.sh
```

Comment:
why no more test?

Reply:
Oh wait, because we actually want to perform the tests now?

Reply:
Yeah we include the test step only when we actually want to run tests.